### ISSUE TITLE
Missing BlockDecoration Class Crash

### ISSUE THREAD

**Issue by DinixM** [rwtema/extrautilities_old_issues#704]
Hello, RWTema. First off, I love the mod, been using it since I first came across it. But, rather recently, after a data wipe/ fresh install of Minecraft, your Extra Utilities mod has started crashing my game on loadup. I can assure that Optifine is not the issue, as it crashes with or without. Without further ado, a crash log on PasteBin for your perusal. 
http://pastebin.com/V28CgZeE
From what I can tell, a BlockDecoration class is missing somehow. How in the name of Notch that happened is anyone's guess. Well, here's hoping for a quick resolution. Thank you for your time and this awesome mod.

EDIT: After some chatting on IRC with a helpful person, I'll be posting up an FML log related to the crash as well. 
http://pastebin.com/ZQ32xndj
Hope this issue is resolved soon! (Or at least, someone comments.)

FINAL EDIT: Thanks to that very same IRC helper, this issue can be closed. As it turns out, the problem was related to Redstone Flux and COFH's API. The fix to this issue is to install cofhlib to force the most recent API version. A big thanks to Temia on Vaskii's IRC for helping me resolve this issue.

TL;DR: If you get this crash, install COFHLib.
